card/lib/card/content/chunk.rb

```diff
@@ -89,6 +89,7 @@ def validate_chunk_list_key chunk_list_key
       register_list :nest_only, [:Nest]
       register_list :query, [:QueryReference]
       register_list :stub, [:ViewStub]
+      register_list :references_keep_escaping,  %i[KeepEscapedLiteral Nest Link]
```

Comment:
I don't love this nomenclature, but I have now failed more than once to come up with anything better.